lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -38,8 +37,9 @@ async function runA11yChecks() {
         'wcag2aa',
       ],
     },
-    resultTypes: ['violations', 'inapplicable'],
+    resultTypes,
```

Comment:
over in https://github.com/GoogleChrome/lighthouse/pull/12536#issuecomment-864454874 we also talked about the value of having `passes` information in the artifact as well. Could start with 
`passes: axeResults.passes.map(result => ({id: result.id})),` like `notApplicable` currently does to get started on that for minimal artifact cost, apparently no additional axe time (since `resultTypes` wasn't limiting that anyways), and it's forward compatible if we ever need to do the full `createAxeRuleResultArtifact` on `passes` since it'll still be extending `{id: string}`.

That would mean only `opts.colorContrastEnabled` would have to be custom between the two things (and maybe wouldn't need a separate method unless folks really want it).